File: src/main/java/com/pengrad/telegrambot/model/request/InputContactMessageContent.java
package com.pengrad.telegrambot.model.request;

/**
 * Stas Parshin
 * 06 May 2016
 */
public class InputContactMessageContent extends InputMessageContent {

    private String phone_number;
    private String first_name;
    private String last_name;

    public InputContactMessageContent(String phoneNumber, String firstName) {
        this.phone_number = phoneNumber;
        this.first_name = firstName;
    }

    public InputContactMessageContent lastName(String lastName) {
        this.last_name = lastName;
        return this;
    }
}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/InputContactMessageContent.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 06 May 2016
 */
class InputContactMessageContent(private val phone_number: String, private val first_name: String) : InputMessageContent(), Serializable {
    private var last_name: String? = null
    private var vcard: String? = null

    fun lastName(lastName: String): InputContactMessageContent {
        this.last_name = lastName
        return this
    }

    fun vcard(vcard: String): InputContactMessageContent {
        this.vcard = vcard
        return this
    }

    companion object {
        private const val serialVersionUID = 0L
    }
}
